Bang.Tests/SaloonSpecification.cs

```diff
@@ -0,0 +1,106 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Bang.Characters;
+using Bang.Game;
+using Bang.Players;
+using Bang.PlayingCards;
+using FluentAssertions;
+using Xunit;
+
+using static Bang.Game.GamePlayInitializer;
+
+namespace Bang.Tests
+{
+    public class SaloonSpecification
+    {
+        [Fact]
+        public void After_played_saloon_card_goes_to_discard_pile()
+        {
+            var gameplay = InitGame();
+            (Player actor, BangGameCard saloonCard) = ChoosePlayer(gameplay);
+
+            // act
+            actor.PlayCard(saloonCard);
+            
+            // Assert
+            gameplay.GetTopCardFromDiscarded().Should().Be(saloonCard);
+        }
+        
+        [Fact]
+        public void Player_discards_gatling_card_after_it_played()
```

Comment:
Copypaste victim :) Not gatling but saloon

Reply:
Renamed :-)